phangsPipeline/casaVisRoutines.py

```diff
@@ -394,47 +420,61 @@ def contsub(
     # find_spw_channels_for_lines
 
     spw_flagging_string = spw_string_for_freq_ranges(
-        infile = infile, 
-        freq_ranges_ghz = ranges_to_exclude,
-        fail_on_empty = True,
+        infile=infile, freq_ranges_ghz=ranges_to_exclude,
+        # fail_on_empty=True)
         )
 
-    if spw_flagging_string is None:
-        logger.error("All data are masked in at least one spectral window. Returning.")
-        return()
+    # <JS> Not sure why one would want fail_on_empty=True here...
```

Comment:
Agreed. Highlighting here as a check for others to review.

Reply:
For this (and the one below), I agree with @astrojysun 's edits. Happy to remove these commented lines and call it a day here

Reply:
👍